Submitty/Submitty
# [Refactor:Router] New URL for SimpleGraderController

### URL Changes

- GET `http://192.168.56.111/f19/sample/gradeable/grading_lab/grading?view=all` Grading Page
- GET `http://192.168.56.111/f19/sample/gradeable/grading_lab/grading/print?section=1&section_type=registration_section&sort=last` Printing Page for Lab
- POST `http://192.168.56.111/f19/sample/gradeable/grading_test/grading/csv` Uploading CSV
- POST `http://192.168.56.111/f19/sample/gradeable/grading_lab/grading` Saving Scores

### Testing
Manually tested the following:
- Go to grading page, try different sort orders, and go to printing page (for lab) check if the ordering is correct.
- Try to upload CSV files for numeric gradeable.
- Try to grade both lab and numeric gradeables.
